patches/tModLoader/Terraria/release_extras/DedicatedServerUtils/manage-tModLoaderServer.sh

```diff
@@ -38,230 +36,251 @@ function update_script {
 		echo "No new script updates"
 	fi
 
-	exit 0
+	exit
+}
+
+function verify_steamcmd {
```

Comment:
Please add more comments to this function.

Consider rename to 'VerifyDownloadToolDependencies' or something to acknowledge ZIP